api/_globals/fetch.json

```
@@ -384,6 +384,42 @@
             "deprecated": false
           }
         }
+      },
+      "stream_body_support": {
+        "__compat": {
+          "description": "Support for `ReadableStream` as body",
+          "spec_url": "https://fetch.spec.whatwg.org/#concept-body-stream",
+          "support": {
+            "chrome": {
+              "version_added": "105"
+            },
+            "chrome_android": "mirror",
+            "edge": "mirror",
+            "firefox": {
+              "version_added": false,
+              "impl_url": "https://bugzil.la/1387483"
+            },
+            "firefox_android": "mirror",
+            "ie": {
+              "version_added": false
+            },
+            "oculus": "mirror",
+            "opera": "mirror",
+            "opera_android": "mirror",
+            "safari": {
+              "version_added": "11.1"
```

Just tested in Safari 18.2 and isn't actually supported:

```suggestion
              "version_added": false
```